tests/proj/simple/test_runscript.py

```diff
@@ -0,0 +1,31 @@
+import os
+import pytest
+
+import jiant.utils.python.io as py_io
+from jiant.proj.simple import runscript as run
+import jiant.scripts.download_data.runscript as downloader
+
+
```

Comment:
Mind adding a comment here (or documentation elsewhere) for what command to use to run this?

Reply:
This is run by default with the usual `pytest` command. No special command is necessary. What are your thoughts on what needs to be clarified?

Reply:
I see, I got confused with the slow tests.